From: Christer Stenbrenden <christer@uggwar.net>
Date: Tue, 25 Feb 2014 22:14:13 +0100
Subject: Don't crash if pa_stream_peek returns NULL

---
 src/mainwindow.cc | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/mainwindow.cc b/src/mainwindow.cc
index be6fb93..3e69af0 100644
--- a/src/mainwindow.cc
+++ b/src/mainwindow.cc
@@ -483,8 +483,8 @@ static void read_callback(pa_stream *s, size_t length, void *userdata) {
     const void *data;
     double v;
 
-    if (pa_stream_peek(s, &data, &length) < 0) {
-        show_error(_("Failed to read data from stream"));
+    if (!data) {
+        pa_stream_drop(s);
         return;
     }
 
