From 81234c49c1b881b6b84cd5df935c0891afc27043 Mon Sep 17 00:00:00 2001
From: Sjoerd Simons <sjoerd@luon.net>
Date: Sat, 26 Apr 2008 00:35:24 +0200
Subject: [PATCH] Define missing flags

---
 src/pavucontrol.cc |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/src/pavucontrol.cc b/src/pavucontrol.cc
index df734f1..a7d9314 100644
--- a/src/pavucontrol.cc
+++ b/src/pavucontrol.cc
@@ -38,6 +38,11 @@
 #define GLADE_FILE "pavucontrol.glade"
 #endif
 
+/* Only defined in the glitch-free branch */
+#define PA_SINK_DECIBEL_VOLUME 32
+#define PA_SOURCE_DECIBEL_VOLUME 32
+
+
 static pa_context *context = NULL;
 static int n_outstanding = 0;
 static bool show_decibel = true;
